File: examples/src/Charts/ValidateChartLayoutAdded.cs
using System.IO
import aspose.slides as slides
using Aspose.slides.Charts
using Aspose.slides.Export
import aspose.pydrawing as drawing

namespace Aspose.slides.Examples.CSharp.Charts
{
    public class ValidateChartLayoutAdded
    {
        public static void Run()
        {
            #ExStart:ValidateChartLayoutAdded
            # The path to the documents directory.
            dataDir = RunExamples.GetDataDir_Charts()
            using (Presentation pres = new Presentation(dataDir+"test.pptx"))
            {
                Chart chart = (Chart)pres.slides[0].shapes.add_chart(slides.charts.ChartType.CLUSTERED_COLUMN, 100, 100, 500, 350)
                chart.validate_chart_layout()
                double x = chart.plot_area.ActualX
                double y = chart.plot_area.ActualY
                double w = chart.plot_area.ActualWidth
                double h = chart.plot_area.ActualHeight


                # Saving presentation
                pres.save(dataDir + "Result.pptx", slides.export.SaveFormat.PPTX)
            }
          

            
            #ExEnd:ValidateChartLayoutAdded
        }
    }
}

File: examples/src/Charts/ValidateChartLayoutAdded.py
import aspose.slides as slides
import aspose.pydrawing as drawing

def charts_validate_chart_layout():
    #ExStart:ValidateChartLayoutAdded
    # The path to the documents directory.
    outDir = "./examples/out/"
    with slides.Presentation() as pres:
        chart = pres.slides[0].shapes.add_chart(slides.charts.ChartType.CLUSTERED_COLUMN, 100, 100, 500, 350)
        chart.validate_chart_layout()
        x = chart.plot_area.actual_x
        y = chart.plot_area.actual_y
        w = chart.plot_area.actual_width
        h = chart.plot_area.actual_height

        # Saving presentation
        pres.save(outDir + "charts_validate_chart_layout_out.pptx", slides.export.SaveFormat.PPTX)
    #ExEnd:ValidateChartLayoutAdded